spec/inferno/dsl/input_output_handling_spec.rb

```diff
@@ -21,6 +21,24 @@
       expect(group.available_inputs.length).to eq(2)
       expect(group.available_inputs.values.map(&:name)).to eq(['b', 'c'])
     end
+
+    it 'does not duplicate renamed inputs' do
```

Comment:
I'm having a hard time figuring out how this is testing duplication.

Reply:
Check out the first commit in this branch (`c517ac95fbdae7a9b022414e72fd4aa497db0a79`) and you'll see how this test fails.

Reply:
Ooo I see now. Thank you.